test/features/composability/forwarder/output/forwarder-dbg.mxsc.json

```diff
@@ -0,0 +1,1523 @@
+{
```

Comment:
`*dbg.mxsc.json` is here by accident, it's not used in tests
call a sc-meta all clean before running the test copy tool